test/integration/gh-63.js

```diff
@@ -0,0 +1,27 @@
+var runWebdriver = require('../run-webdriver');
+var AxeBuilder = require('../../lib');
+
+describe('gh-63: buy.garmin.com/en-US/US/p/591046', function() {
+  this.timeout('1m');
+
+  var driver;
+  before(function() {
+    driver = runWebdriver();
+    driver
+      .manage()
+      .timeouts()
+      .setScriptTimeout(5000);
+    return driver.get('https://buy.garmin.com/en-US/US/p/591046');
```

Comment:
This seems more than a little fragile. Can we do something to reproduce the problem somewhere we control ourselves instead?

Reply:
I have not successfully reproduced this outside of the Garmin page. I _think_ we'd have to create a Facebook "like button" and serve some HTML containing it over HTTPS, which seems like a lot of work.

Instead, should I put this test behind a flag (so it's conditionally run)?

Reply:
Flag sounds good.